chef/cookbooks/hyperv/recipes/do_ceilometer.rb

```diff
@@ -0,0 +1,8 @@
+raise if not node[:platform] == 'windows'
+
+include_recipe "hyperv::openstack_install_ceilometer"
+include_recipe "hyperv::openstack_install_neutron"
+include_recipe "hyperv::config_neutron"
+include_recipe "hyperv::config_ceilometer"
+include_recipe "hyperv::register_services_ceilometer"
+include_recipe "hyperv::register_services_neutron"
```

Comment:
One other thought: does it make sense to install ceilometer on hyperv nodes without installing also nova? And if nova is already there, it requires neutron, so ceilometer wouldn't need to require neutron, I think